python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -825,7 +825,7 @@ def _outputZeropoints(self, butler, zptCat, visitCat, offsets, tract=None):
 
             # The "mean" calibration will be set to the center of the ccd for reference
             calibCenter = fgcmField.evaluate(fgcmField.getBBox().getCenter())
-            calibErr = (np.log(10.0) / 2.5) * calibCenter * rec['fgcmZptErr']
+            calibErr = (np.log(10.0) / 2.5) * calibCenter * np.sqrt(rec['fgcmZptVar'])
```

Comment:
Another gentle reminder of https://developer.lsst.io/python/style.html#binary-operators-should-be-surrounded-by-a-single-space-except-for (although I think there’s much inconsistency even within this file, so do as you wish... 😉)